charter/index.html

```diff
@@ -598,11 +614,12 @@ <h3>
 
     <footer>
         <address>
-            <a href="mailto:???@w3.org">??? (LINK TBD)</a>
+            <a class="issue" href="mailto:???@w3.org">??? (LINK TBD)</a>
```

Comment:
I believe this should be `class="todo"` item as I don't believe we'll have an official email address until the working group process completes.

Reply:
+1

Reply:
```suggestion
            <a class="todo" href="mailto:???@w3.org">??? (LINK TBD)</a>
```